submissions/AndriiBabushko/html-css_popup/styles/sass/fonts.sass

```diff
@@ -0,0 +1,2 @@
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono&display=swap') // 'Space Mono'
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap') // 'Dancing Script'
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

Reply:
✓